app/index.html

```diff
@@ -131,6 +131,58 @@
     </div>
   </ui-minimize-panel>
 
+  <ui-minimize-panel title="File list" open='false'>
```

Comment:
Should this display after that Analysis pane? It seems to be later in the workflow (populated from Analysis).